apps/prairielearn/src/lib/editors.ts

```diff
@@ -1125,6 +1128,9 @@ export class QuestionTransferEditor extends Editor {
     debug(`Write info.json with new title and uuid`);
     infoJson.title = questionTitle;
     infoJson.uuid = this.uuid;
+    delete infoJson['sharingSets'];
+    delete infoJson['publiclyShared'];
+    delete infoJson['publiclySharedWithSource'];
```

Comment:
this should solve our problem nicely for people copying public questions into their own course